images/windows/scripts/build/Install-AzureCli.ps1

```diff
@@ -15,7 +15,7 @@ New-Item -ItemType 'Directory' -Path $azureCliExtensionPath | Out-Null
 
 Install-Binary -Type MSI `
     -Url 'https://aka.ms/installazurecliwindowsx64' `
-    -ExpectedSignature 'F9A7CF9FBE13BAC767F4781061332DA6E8B4E0EE'
+    -ExpectedSignature '245D262748012A4FE6CE8BA6C951A4C4AFBC3E5D'
```

Comment:
Ideally this should be a separate PR. These changes have nothing to do with the WDK.